core/common/src/main/java/alluxio/retry/ExponentialTimeBoundedRetry.java

```diff
@@ -24,29 +25,34 @@
  * timings would be [10, 20, 40, 80, 100, 100, 100, 50], assuming the operation being retries takes
  * no time. The 50 at the end is because the previous times add up to 450, so the mechanism sleeps
  * for only 50ms before the final attempt.
+ *
+ * However, those are just the base sleep timings. For each sleep time, we multiply by a random
+ * number from 1 to 1.1 to add jitter to avoid hotspotting.
  */
-public final class TimeBoundedExponentialRetry extends TimeBoundedRetry {
+public final class ExponentialTimeBoundedRetry extends TimeBoundedRetry {
   private final Duration mMaxSleep;
   private Duration mNextSleep;
 
   /**
    * See {@link Builder}.
    */
-  private TimeBoundedExponentialRetry(TimeContext timeCtx, Duration maxDuration,
-      Duration initialSleep, Duration maxSleep) {
+  private ExponentialTimeBoundedRetry(TimeContext timeCtx, Duration maxDuration,
+                                      Duration initialSleep, Duration maxSleep) {
```

Comment:
Formatting

Reply:
apparently IntelliJ will reformat the constructor when you rename a class... >.<